.gitignore

```diff
@@ -11,3 +11,4 @@ tcp_server
 template.tar.gz
 .*.swp
 test
+venv
```

Comment:
should this be included? the venv normally only exists within the container and not on the host machine

Reply:
the venv is needed dev-side for lsp/flake/etc